### PR TITLE
Query with an input file

### PR DESCRIPTION
**Pull request by lisabdj**
Hello everyone,

I open this PR for code update to make queries with an input file.

The input file is written in YAML and is read in Python in the script scrap_zenodo.py. This input file contains the list of keywords related to molecular dynamics, generic keywords for zip archives and information about each filetype.

### CODE REVIEW

File: scrap_zenodo.py [hunk 1]
```diff
@@ -5,16 +5,58 @@
...
+    md_keywords = data_loaded["md_keywords"]
+    generic_keywords = data_loaded["generic_keywords"]
+    file_types = data_loaded["file_types"]
+    return md_keywords, generic_keywords, file_types
+
+
+STRING_MD_KEYWORDS = "AND (" + md_keywords[0]
+for i in range(1, len(md_keywords)):
+    STRING_MD_KEYWORDS += " OR " + md_keywords[i]
+STRING_MD_KEYWORDS += ")"
```
Comment (pierrepo) [inline scrap_zenodo.py:52]: Could you check if something like this would work?
```
MD_KEYWORDS = "AND (" + " OR ".join(md_keywords) + ")"
```

Comment (pierrepo) [reply scrap_zenodo.py:52]: Many thanks @lisabdj 
Out of curiosity, how many datasets and files do you get with this yaml file?

Comment (lisabdj) [reply scrap_zenodo.py]: With this yaml file, without the zip query, I get 1126 datasets. Indeed, with the zip query, I get the following message: {'status': 400, 'message': 'Maximum number of 10000 results have been reached.'}. The query stops after 10 000 datasets found.

I counted the number of datasets and files found by filetype, as well as the total datasets (without the zip query and without the duplicates) in the following google sheet : https://docs.google.com/spreadsheets/d/1TTVhbk0SR0wtqR72jSh_JcmcosItA9z1lISXGyIFvBU/edit?usp=sharing

**Comment by lisabdj** [comment]
I have a problem with the scrap_zip_content() function. There is an error message when the function scraps the link https://zenodo.org/record/802915/preview/cluster-all.zip while I have no problem when the script parser_web.py reads this link. 
The error message I get on the scrap_zip_content function is: AttributeError: **'NoneType' object has no attribute 'findAll'**.

**Comment by lisabdj** [comment]
I forgot to mention: I have this error at line 175 "for row in table.findAll("span"):"

**Comment by pierrepo** [comment]
> I have a problem with the scrap_zip_content() function. There is an error message when the function scraps the link https://zenodo.org/record/802915/preview/cluster-all.zip while I have no problem when the script parser_web.py reads this link.
The error message I get on the scrap_zip_content function is: AttributeError: 'NoneType' object has no attribute 'findAll'.

The above link points to an incomplete preview. The first 100 files only are listed. This is a known limitation from Zenodo. We'll try catch the error.

**Pull request merged**